lib/jsonapi_compliable/errors.rb

```diff
@@ -11,6 +65,36 @@ def initialize(validation_response)
       end
     end
 
+    class ModelNotFound < Base
+      def initialize(resource_class)
+        @resource_class = resource_class
+      end
+
+      def message
+        <<-MSG
+Could not find model for Resource '#{@resource_class}'
```

Comment:
I'd also add a reference to the exact model we were looking for as well, for added clarity.

Reply:
I think this is probably not possible, since there is both a lookup and a fallback. I think the below message of "if it does not match the name of the resource" is OK for now at least